sql/auth/sql_auth_cache.cc

```diff
@@ -1166,6 +1174,17 @@ ulong acl_get(THD *thd, const char *host, const char *ip, const char *user,
     }
   }
 
+  /* Check to see if the inquiry is for the utility_user */
+  if (acl_is_utility_user(user, host, ip)) {
+    /* Check to see if database is within the schema access list */
+    std::vector<std::string>::const_iterator it =
```

Comment:
```suggestion
    auto it =
```

Reply:
Probably needs s/begin()/cbegin() s/end()/cend() below too, so that it's const_iterator and not iterator